package.json

```diff
@@ -91,7 +91,7 @@
     ]
   },
   "lint-staged": {
```

Comment:
Is this a script, or it's own separate thing?

Reply:
- https://github.com/okonet/lint-staged
  - https://github.com/okonet/lint-staged#configuration
    - > Starting with v3.1 you can now use different ways of configuring it:
      - > `lint-staged` object in your `package.json`
      - > `.lintstagedrc` file in JSON or YML format, or you can be explicit with the file extension: `.lintstagedrc.json`
      - > `.lintstagedrc.yaml`
      - > `.lintstagedrc.yml`
      - > `lint-staged.config.js`, `.lintstagedrc.js`, or `.lintstagedrc.cjs` file in JS format

All good!

[OOS/Future] could potentially be nice to move it to one of those file types like `.lintstagedrc` or `lint-staged.config.js` (my personal preference)

Reply:
This part is crucial - it's a performance optimization so husky only runs over the files staged in the index, not the whole repo. Makes commits go from 5-7 seconds to a few milliseconds. Not really related to the linting, but good catch.